src/java/picard/analysis/GcBiasMetricsCollector.java

```diff
@@ -125,13 +126,12 @@ public void acceptRecord(final GcBiasCollectorArgs args) {
             final SAMRecord rec = args.getRec();
             final String type;
             if (!rec.getReadUnmappedFlag()) {
-                final ReferenceSequence ref = args.getRef();
-                final byte[] refBases = ref.getBases();
-                StringUtil.toUpperCase(refBases);
-                final int refLength = refBases.length;
-                final int lastWindowStart = refLength - scanWindowSize;
-
-                if(referenceIndex != rec.getReferenceIndex() || gc == null){
+                if(referenceIndex != rec.getReferenceIndex() || gc == null || refBases == null){
```

Comment:
I was reading too fast.  I was thinking you meant that the refBases == null check wasn't necessary (since it's not).  But that doesn't really matter I guess.